fulp_modules/features/species/beefmen/beefman_clothing.dm

```diff
@@ -175,64 +175,64 @@
 
 /obj/item/clothing/under/bodysash/bartender
 	name = "bartender's sash"
-	desc = "A simple bartender sash without any flair, slung from shoulder to hip."
+	desc = "A humble bartender's sash slung from shoulder to hip."
 	icon_state = "bartender"
 
 /obj/item/clothing/under/bodysash/chaplain
 	name = "chaplain's sash"
-	desc = "A simple religious sash, slung from shoulder to hip."
+	desc = "A simple religious sash slung from shoulder to hip."
 	icon_state = "chaplain"
 
 /obj/item/clothing/under/bodysash/curator
 	name = "curator's sash"
-	desc = "A simple librarian sash, slung from shoulder to hip."
+	desc = "An unstylish librarian's sash slung from shoulder to hip."
 	icon_state = "curator"
 
 /obj/item/clothing/under/bodysash/lawyer
 	name = "lawyer's sash"
-	desc = "A simple lawyer sash, slung from shoulder to hip."
+	desc = "A formal lawyer's sash slung from shoulder to hip."
 	icon_state = "lawyer"
 
 /obj/item/clothing/under/bodysash/botanist
 	name = "botanist's sash"
-	desc = "It's a sash designed to protect against minor plant-related hazards."
+	desc = "A sash designed to protect against minor, plant-related hazards."
```

Comment:
I don't think this comma is supposed to be here?

Reply:
True, I'll fix that.